src/Resources/envs/symfony/defaults.yml

```diff
@@ -37,7 +37,7 @@ packages:
         enabled:    false
         required:   false
         default:    '5'
```

Comment:
should we make it the default version?

Reply:
I don't see why we couldn't :) I've made the change